tuxemon/mod_manager/__init__.py

```diff
@@ -198,12 +223,24 @@ def remove_package_from_list(self, name):
                 json.dumps(before, indent=4)
             )
 
-
     def remove_package(self, name):
         """Removes the local package"""
         # Get the path
         path = self.read_package_from_list(name)
+        if os.path.isabs(path):
+            raise IOError("Path is absolute")
         if path != sanitize_paths(path):
             raise ValueError("Detected incorrect characters in path")
         shutil.rmtree(path, ignore_errors=True)
         self.remove_package_from_list(name)
+
+    def install_local_package(self, filename, name=None, download_deps=False, link_deps=False):
```

Comment:
is this used anywhere?

Reply:
ok.  is this finished then, or will be used in the future?  generally, better to avoid adding code if its not being used.